### PR TITLE
Drewtest3

### PR DESCRIPTION
**Pull request by drewwilliams5280**

### CODE REVIEW

File: code/api/src/modules/user/resolvers.js [hunk 1]
```diff
@@ -28,19 +28,26 @@ export async function create(parentValue, { name, email, password }) {
...
         secondaryStyle
       },
       { where: { id } }
     )
-  // } else {
-  //   throw new Error('Operation denied.')
-  // }
+  } else {
+    throw new Error('Please login first to update information.')
+  }
```
Comment (LHJE) [inline code/api/src/modules/user/resolvers.js:50]: Great fix!

**Review by LHJE — APPROVED** [review]
Nicely!

File: code/api/src/modules/product/types.js [hunk 2]
```diff
@@ -14,6 +14,7 @@ const ProductType = new GraphQLObjectType({
     style: { type: GraphQLString },
     gender: { type: GraphQLInt },
     description: { type: GraphQLString },
+    style: { type: GraphQLString },
```
Comment (Arique1104) [inline code/api/src/modules/product/types.js:17]: Good catch making sure this was present.

File: code/api/src/modules/user/resolvers.js (hunk 1 above)
Comment (Arique1104) [reply code/api/src/modules/user/resolvers.js:50]: Glad we could isolate the issues in this code.  So glad that we know our code is working!

**Review by Arique1104 — APPROVED** [review]

**Pull request merged**